CHANGELOG.d/breaking_remove-two-ad-hoc-case-statements.md

```diff
@@ -0,0 +1,20 @@
+* Remove ad-hoc same-line single-branch case statements
```

Comment:
It's not single line case statements. Single line case statements are ok. It's a particular form of non-single-line case statements when there's only one branch.

Reply:
Also, a nit, but they are case expressions, not statements.

Reply:
I've reworked the entry. How about now?